Lib9c/Action/DPoS/Model/RedelegationEntry.cs

```diff
@@ -26,10 +26,12 @@ public class RedelegationEntry : IEquatable<RedelegationEntry>
             Address = DeriveAddress(redelegationAddress, index);
             RedelegationAddress = redelegationAddress;
             RedelegatingShare = redelegatingShare;
+            InitialConsensusToken = unbondingConsensusToken;
             UnbondingConsensusToken = unbondingConsensusToken;
             IssuedShare = issuedShare;
             Index = index;
             CompletionBlockHeight = blockHeight + UnbondingSet.Period;
```

Comment:
Unbonding period have to be maintained with protective manner, to find effective block height easily for production, IMO.

Reply:
I think we should work on `Unbonding period` in a different PR.